pyproject.toml

```diff
@@ -3,8 +3,6 @@
 requires = [
     "setuptools",
     "wheel",
-    "Cython>=0.29.33",
```

Comment:
Don't we need to keep at least `"Cython"`?

Reply:
Oops indeed